docs/sphinx/source/whatsnew/v0.9.5.rst

```diff
@@ -26,13 +26,14 @@ Enhancements
   differentiable and bounded between zero and one. (:pull:`1179`)
 * Add ``model='gueymard2003'``, the airmass model used for REST and REST2,
   to :py:func:`~pvlib.atmosphere.get_relative_airmass`. (:pull:`1655`)
-
-
 * Added an optional ``model`` parameter to
   :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance` and
   :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance_poa`
   to enable use of the hay-davies sky diffuse irradiance model
   instead of the default isotropic model. (:pull:`1668`)
+* Use [Horner's Method](https://en.wikipedia.org/wiki/Horner%27s_method)
```

Comment:
This markdown link won't render correctly in an RST file (I make this mistake all the time)

Reply:
Right on both accounts! Thanks

> is there supposed to be a new asv benchmark included in the PR

I wasn’t planning on it, but I did wait until the `time_disc()` benchmark was implemented before marking this for review. I also noticed there’s not a benchmark for GTI-DIRINT, which would benefit from this PR

Reply:
d'oh, I forgot disc already has one on `main`!  thanks